contracts/src/Utilities/README.md

```diff
@@ -0,0 +1,18 @@
+## Utility Contracts
+
+### ValidatorSetSig Contract
+
+This contract provides an alternative to traditional multi-signature contracts where instead of requiring signatures from `K` of `N` pre-specified signers, an aggregate signature is required from a quorum of the current validators a given blockchain.
```

Comment:
```suggestion
This contract provides an alternative to traditional multi-signature contracts where instead of requiring signatures from `K` of `N` pre-specified signers, an aggregate signature is required from a quorum of the current validators for a given blockchain.
```